content/posts/python-type.md

```diff
@@ -0,0 +1,143 @@
++++
+title = "Objects in Python"
+date = 2023-04-03
+tags = ["python"]
++++
+
+Everything in Python has a type (even the type `type`!) because everything is an object.
+
+Objects are abstraction of data in Python.
+All data in a Python program is stored as a object.
+
+An object allows instances of it to be created.
+The created instances, which are also objects are destroyed by garbage collection.
```

Comment:
Maybe reword as 'instances can be created from objects.' I'm not sure what you mean by 'instances are destroyed by garbage collection'. Everything in python is an object which _can_ be destroyed by garabge collections if there are no references to it.

Reply:
reworded it, I don't wish to mention `reference counting` as it will take the reader away - I will cover it in a future post.